src/main/java/com/google/api/generator/gapic/composer/utils/RetrySettingsUtils.java

```diff
@@ -0,0 +1,68 @@
+// Copyright 2022 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.api.generator.gapic.composer.utils;
+
+import com.google.api.generator.engine.ast.MethodInvocationExpr;
+import com.google.api.generator.engine.ast.PrimitiveValue;
+import com.google.api.generator.engine.ast.TypeNode;
+import com.google.api.generator.engine.ast.ValueExpr;
+import com.google.api.generator.gapic.composer.store.TypeStore;
+import com.google.protobuf.Duration;
+import com.google.protobuf.util.Durations;
+import java.util.Arrays;
+import java.util.List;
+
+/** Common utility methods to expose */
+public class RetrySettingsUtils {
```

Comment:
I know these methods are only used by retry related logics, but they can be used by other features as well, can we have a more generic name for this class? 
I'm bad at naming as well, something like `CommonUtils` seems too generic but something like `DurationUtils` seems inaccurate. Maybe break it further into two utils? One is `ValueExprUtils`, another one is  `MethodInvocationExprUtils`?

Reply:
How about `ConvertToExprUtils` as updated below? I tried to keep it generic in case of future additions. Is it too generic? I also don't want split into two classes as the `createDurationOfMillisExpr()` is clearly close related to the `toValExpr` ones.